addons/website/static/tests/tours/media_dialog.js

```diff
@@ -143,16 +143,26 @@ registerWebsitePreviewTour("website_media_dialog_image_shape", {
         name: "Text - Image",
         groupName: "Content",
     }),
+    {
+        // This step is required to ensure the image loads correctly.
+        content: "Click on the text image",
+        trigger: ":iframe .s_text_image",
+        run: "click",
+    },
```

Comment:
why not use clickOnSnippet helper function? As next step is to click on img then why we need this step?